airflow/dag_processing/processor.py

```diff
@@ -159,6 +161,7 @@ class DagFileParseRequest(BaseModel):
     """
 
     file: str
+    bundle_path: Path
```

Comment:
It might be worth adding some text around file vs bundle_path, and why we need both.

Reply:
New thought: Is it easier to just pass around relative path in the first place?

Reply:
i considered it but this way was easier; it may change in future. it's internal code